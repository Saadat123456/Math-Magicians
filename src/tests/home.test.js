import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../pages/home';

it('renders correctly', () => {
  const tree = renderer
    .create(<Home ></Home>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});