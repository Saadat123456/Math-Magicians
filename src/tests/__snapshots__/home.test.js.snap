// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main
  className="home mainStyle"
>
  <div
    className="details"
  >
    <h2
      className="heading"
    >
      Welcome To Our Page
    </h2>
    <div
      className="description descripition-home"
    >
      <div>
        <p
          className="paragraph"
        >
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellat debitis similique excepturi, a sapiente reiciendis illo eos ipsam dolorem natus unde architecto magnam, dolore nostrum id. Ipsa accusamus molestias soluta! Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellat debitis similique excepturi, a sapiente reiciendis illo eos ipsam dolorem natus unde architecto magnam, dolore nostrum id. Ipsa accusamus molestias soluta!
        </p>
        <p
          className="paragraph"
        >
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellat debitis similique excepturi, a sapiente reiciendis illo eos ipsam dolorem natus unde architecto magnam, dolore nostrum id. Ipsa accusamus molestias soluta! Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellat debitis similique excepturi, a sapiente reiciendis illo eos ipsam dolorem natus unde architecto magnam, dolore nostrum id. Ipsa accusamus molestias soluta!
        </p>
      </div>
    </div>
  </div>
  <span />
</main>
`;
